src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -34,8 +34,8 @@
 {
     static String stacksTokenVerificationBaseUrl;
     static String stacksUrlErrorMsg = "rejecting due to invalid stacksURL property (" + stacksTokenVerificationBaseUrl + ")";
-    static final String HONEYBADGER_KEY = "WOWZA_HONEYBADGER_API_KEY";
-    static final String HONEYBADGER_ENVIRONMENT = "WOWZA_ENVIRONMENT";
+    static final String HONEYBADGER_KEY_ENV_VAR = "WOWZA_HONEYBADGER_API_KEY";
```

Comment:
ok, the names are less similar, though i went with your earlier suggestion for the API key, and something slightly different than the above suggestion for the env name (so `HONEYBADGER_API_KEY_ENV_VAR`, and `HONEYBADGER_ENV_NAME_ENV_VAR`, respectively).

i do like the `HONEYBADGER_*_ENV_VAR` naming convention in general, since they're honeybadger related system environment variables, but hopefully this addresses the `ENV_ENV` concern (which made sense), and hopefully the immediate differences after `HONEYBADGER_` make the names easier to differentiate at a glance.